src/main/java/frc/robot/subsystems/flywheel/FlywheelComponents.java

```diff
@@ -0,0 +1,21 @@
+package frc.robot.subsystems.flywheel;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.robot.hardware.motor.ControllableMotor;
+import frc.robot.hardware.request.IRequest;
+import frc.robot.hardware.signal.InputSignal;
+
+public record FlywheelComponents(
```

Comment:
consider stuff/avatiach

Reply:
no

Reply:
Until a concrete decision is made, any name would be acceptable (unless you misuse your power)